exercises/complex-numbers/test/Tests.hs

```diff
@@ -12,14 +12,15 @@ main = hspecWith defaultConfig {configFastFail = True} specs
 
 specs :: Spec
 specs = do
-          describe "real"      $ for_ realCases      $ testC real
-          describe "imaginary" $ for_ imaginaryCases $ testC imaginary
-          describe "conjugate" $ for_ conjugateCases $ testB conjugate
-          describe "abs"       $ for_ absCases       $ testC abs
-          describe "mul"       $ for_ mulCases       $ testA mul
-          describe "div"       $ for_ divCases       $ testA div
-          describe "add"       $ for_ addCases       $ testA add
-          describe "sub"       $ for_ subCases       $ testA sub
+          describe "real"       $ for_ realCases       $ testC real
```

Comment:
since it's no longer necessary to add an extra space to all these lines, I'd say to remove them

Reply:
Done.